udata_front/theme/gouvfr/assets/js/components/search/search.vue

```diff
@@ -173,20 +203,22 @@
 
 <script>
 import { defineComponent, ref, onMounted, computed } from "vue";
-import {useI18n} from 'vue-i18n';
+import { useI18n } from 'vue-i18n';
 import axios from "axios";
 import { generateCancelToken, apiv2 } from "../../plugins/api";
-import {useToast} from "../../composables/useToast";
+import { useToast } from "../../composables/useToast";
 import useSearchUrl from "../../composables/useSearchUrl";
 import SearchInput from "./search-input.vue";
 import Dataset from "../dataset/card-lg.vue";
 import Loader from "../dataset/loader.vue";
 import SchemaFilter from "./schema-filter.vue";
-import Empty from "./empty.vue";
 import { Pagination } from "@etalab/udata-front-plugins-helper";
 import MultiSelect from "./multi-select.vue";
-import { search_autocomplete_debounce } from "../../config";
+import ActionCard from "../Form/ActionCard/ActionCard.vue";
+import { data_search_form_url, search_autocomplete_debounce } from "../../config";
 import { debounce } from "../../composables/useDebouncedRef";
+import searchIcon from "svg/illustrations/search.svg";
+import search2Icon from "svg/illustrations/search2.svg";
```

Comment:
Ahah I wonder if we could have more explicit names for theses? Maybe the second one could be named `magnifying_glass.svg`?